celery_pubsub/pubsub.py

```diff
@@ -84,6 +92,20 @@ def _topic_to_re(topic: str) -> re.Pattern[str]:
 _pubsub_manager: PubSubManager = PubSubManager()
 
 
+def subscribe_to(topic: str) -> typing.Callable:
+    def decorator(
+        func: typing.Callable[..., typing.Any]
+    ) -> typing.Callable[..., typing.Any]:
```

Comment:
```suggestion
    def decorator(func: typing.Callable[P, R]) -> Task[P, R]:
```
I think that's more correct, right?

Reply:
fix them